LICENSE

```diff
@@ -1,29 +1,193 @@
-Copyright (c) 2015, Tristan Sloughter <t@crashfast.com>.
+
 All rights reserved.
 
-Redistribution and use in source and binary forms, with or without
```

Comment:
Since @tsloughter copyrighted this (copyright is still in the file, just at the bottom) he will have to sign off on this. 

Rationale for updating the license is we currently publish to hex with a license type of Apache-2.0, but the license isn't actually that.